uber/site_sections/registration.py

```diff
@@ -153,6 +153,65 @@ def change_badge(self, session, id, message='', **params):
             'attendee': attendee
         }
 
+    def qrcode_generator(self, data):
+        """
+        Takes a piece of data and encrypts it using the QR_CODE_PASSWORD
+        Then, adds the EVENT_QR_ID, and returns an Aztec barcode as an image stream.
+        Args:
+            data: A string to encrypt.
+
+        Returns: A PNG buffer. Use this function in an img tag's src='' to display an image.
+
+        """
```

Comment:
Aaaand I immediately noticed that you ARE setting the ``Content-Type`` header.  MY BAD.